### ISSUE TITLE
Cannot add `EsFieldArgs` without `type` field

### ISSUE THREAD

**Issue by shusson** [xrobert35/es-mapping-ts#3]
Elasticsearch will fail if you add the `type` field and the `enabled` field

Example:
```typescript
export class PostEntity {
    @EsField({ type: "text", enabled: false })
    comment: string;
}
```
ES will complain with something like:
```
"Mapping definition for [comment] has unsupported parameters:  [enabled : false]\"}}
```

Work around:
```typescript
export class PostEntity {
    @EsField({ enabled: false } as any)
    comment: string;
}
```

Maybe we should make `type` in `EsFieldArgs` optional?

**Comment by xrobert35** [comment]
Hum, this is a strange behaviour cause enabled is an official parameter  : https://www.elastic.co/guide/en/elasticsearch/reference/current/enabled.html.

Anyway i'm ok with you.  In the document it look like it's possible to add a field without type

**Comment by xrobert35** [comment]
Fixed in 0.0.14
